### DISCUSSION TITLE
Anyway to pull all successful assertions?

### DISCUSSION THREAD

**Discussion by Rajesh-Narayanappa87** [chaijs/chai#1412] [Support]
Hi Team,
Thanks for giving this wonderful library. Appreciate all efforts. 
I have a query.. I have integrated chai with mocha and on assertion failure, I am getting failed assertion. 
I am finding a way to pull success assertions too. Can you pls help me if this feature is available. 

We have our own custom report. We have a requirement to print successful assertion. I could not find this in documentation. Could you please help

**Comment by keithamus** [comment]
It's not currently available. The plan is for Chai 5 architecture work to allow this sort of thing.
